ui/src/features/Editor/index.tsx

```diff
@@ -51,7 +51,7 @@ export default function Editor({
     handleWorkflowUndo,
     handleWorkflowRename,
   } = useHooks({ yWorkflows, undoManager, undoTrackerActionWrapper });
-
+  const isMainWorkflow = openWorkflows[0]?.id === currentWorkflowId;
```

Comment:
`const isMainWorkflow = currentWorkflowId === DEFAULT_ENTRY_GRAPH_ID`
Please move this in to the useHooks